packages/design-system/src/icons/icon-download.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { IconBase, IconElement, IconProps } from './icon-base';
+
+export const IconDownload = React.forwardRef<IconElement, Readonly<IconProps>>(
+  ({ ...props }, forwardedRef) => (
+    <IconBase ref={forwardedRef} {...props} width={18} height={18}>
+      <path
```

Comment:
this one is the one from react-feather, not sure we have another source of icons tho, let me know

Reply:
I'll put here the download icon

Reply:
```svg
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M4.75 14.75V16.25C4.75 17.9069 6.09315 19.25 7.75 19.25H16.25C17.9069 19.25 19.25 17.9069 19.25 16.25V14.75" stroke="#A09FA6" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
  <path d="M12 14.25V4.75" stroke="#A09FA6" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
  <path d="M8.75 10.75L12 14.25L15.25 10.75" stroke="#A09FA6" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

Reply:
updated!